### PR TITLE
chore: adds tessen configuration

### PR DESCRIPTION
**Pull request by jpvajda**
This PR adds the configuration for tessen and remove the individual GA tracking code so we can leverage all the goodness of Tessen and Segment. 🎸

### CODE REVIEW

**Comment by LizBaker** [comment]
if you're removing ga you can also remove the `onRouteUpdate` function from gatsby-browser.js to keep things clean

**Comment by jpvajda** [comment]
@LizBaker Ok, I see on the Docs Site we handle this a little different for `OnRouteUpdate` Should I take that approach on the developer site in stead of removing it all together? 

https://github.com/newrelic/docs-website/blob/develop/gatsby/on-route-update.js

```
const onRouteUpdate = ({ location, prevLocation }) => {
  if (
    window.newrelic &&
    location.hash &&
    location.pathname !== prevLocation?.pathname
  ) {
    window.newrelic.addPageAction('hash_request', { hash: location.hash });
  }
};
```

**Comment by LizBaker** [comment]
@jpvajda I don't think so. I'm not sure why we're tracking that but since it doesn't have to do with tessen or GA (looks like it's the browser agent) I'd ignore it for this PR

File: gatsby-config.js [hunk 1]
```diff
@@ -16,14 +16,13 @@ module.exports = {
...
       options: {
-        gaTrackingId: 'UA-3047412-33',
         layout: {
           contentPadding: '2rem',
           maxWidth: '1700px',
           component: require.resolve('./src/layouts'),
           mobileBreakpoint: '760px',
         },
-       prism: {
+        prism: {
```
Comment (zstix) [inline gatsby-config.js:25]:
```suggestion
       prism: {
```

File: gatsby-config.js [hunk 2]
```diff
@@ -16,14 +16,13 @@ module.exports = {
     {
       resolve: '@newrelic/gatsby-theme-newrelic',
       options: {
-        gaTrackingId: 'UA-3047412-33',
```
Comment (zstix) [inline gatsby-config.js:19]: Are we positive we're not using this tracking ID for anything else?

Just want to make sure that the addition of Tessen (and Segment) replaces everything we were getting from Google Analytics.

**Review by zstix — COMMENTED** [review]
I had one question and a minor nitpick change. If we're confident that this the path forward, I'm good with it!

File: gatsby-config.js (hunk 2 above)
Comment (jpvajda) [reply gatsby-config.js:19]: according to Zoe we no longer need this in the site config. I looked at the Docs site and how it was [implemented](https://github.com/newrelic/docs-website/wiki/Event-Data--and--Instrumentation)  and the way I understand it to work is: we rely on this configuration with tessen to pass all the data we need to segment and GA, and within Segment we have the GA tracking ID  configured. 

``` tessen: {
          product: 'DOC',
          subproduct: 'TDOC',
          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
          trackPageViews: true,
          pageView: {
            name: 'pageView',
            category: 'DocPageView',
            getProperties: ({ location, env }) => ({
              path: location.pathname,
              env: env === 'production' ? 'prod' : env,
            }),
          },
        },

```

**Review by LizBaker — APPROVED** [review]

**Comment by jpvajda** [comment]
Once I test and confirm this is working with the OSS SIte, I'll cut a release for the developer site.

**Pull request merged**